fwsrc/commonservices.c

```diff
@@ -101,12 +104,12 @@ static void ICACHE_FLASH_ATTR scandone(void *arg, STATUS status)
 	if( !c->pbss ) { scanplace = -1;  return;  }
 	scanplace = 0;
 
-	printf( "ISCAN\n" );
+	os_printf( "ISCAN\n" );
```

Comment:
I modified all printfs to be os_printfs. I read something about this optimizing string memory, but I can't find it now for proof.